modules/rds-aurora/main.tf

```diff
@@ -0,0 +1,85 @@
+variable "azs" {
+	type = list(string)
+	description = "Availibility zones for cluster to use."
+}
+
+variable "db_name" {
+	type = string
+	description = "Database name."
+}
+
+variable "db_password" {
+	type = string
+	description = "Database master password."
+}
+
+variable "db_username" {
+	type = string
+	description = "Database master username."
+}
+
+variable "engine" {
+	type = string
+	description = "Database engine name."
+}
+
+variable "identifier" {
+	type = string
+	description = "Cluster identifier."
+}
+
+variable "instance_class" {
+	type = string
+	default = "db.r4.large"
+	description = "Database instance type."
+}
+
+variable "instance_count" {
+	type = number
+	default = 1
+	description = "Number of database instances to run."
+}
+
+variable "sg_ids" {
+	type = list(string)
+	description = "Security group ids that controls the access to database."
+}
+
+variable "subnet_ids" {
+	type = list(string)
+	description = "Subnet ids for database to live in. Better be private subnets."
```

Comment:
As I remember it from setting up RDS a couple of weeks ago this list is supposed to contain at least 2 elements even for single-az deployment - https://forums.aws.amazon.com/thread.jspa?threadID=151938
What do you think about adding this link as a reminder?

Reply:
Yes, that is true. At least two subnets are required unless there is only one AZ in the region. I will put the link.